tests/carl.py

```diff
@@ -58,7 +58,9 @@
 if __name__ == "__main__":
 
     # Create brain connection
-    log_path = pathlib.Path.cwd().parent / 'src' / 'cltl' / 'brain' / 'logs'
+   # Piek: parent is the folder above the project, had to take it out
+   # log_path = pathlib.Path.cwd().parent / 'src' / 'cltl' / 'brain' / 'logs'
+    log_path = pathlib.Path.cwd() / 'src' / 'cltl' / 'brain' / 'logs'
```

Comment:
Can't we just use a temp folder for this?

Reply:
OK, I made it a command line parameter and use a temporary directory by default (which is cleaned up after running the script).

Reply:
Some warning/error messages while running  `for example in *.py; do python $example; done`

```
Failed to query Wikidata
```

```
Traceback (most recent call last):
  File "jsonld.py", line 19, in <module>
    jsonld_context = brain.get_jsonld_context(save_path=jsonld_filepath)
  File "/home/tk/repos/cltl-knowledgerepresentation/src/cltl/brain/basic_brain.py", line 192, in get_jsonld_context
    with open(save_path, 'w') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/tk/repos/cltl-knowledgerepresentation/cltl/brain/ontologies/integration_context.jsonld'
```

Reply:
haha sorry I forgot to do `git pull`. The `jsonld.py` error doesn't happen anymore.